src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from 'react';
+
+export function Person({ person }) {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age && <p className="Person__age">I am {age}</p>}
+      { isMarried
+        ? (
+          <p className="Person__partner">
+            {partnerName} is my {sex === 'm' ? 'wife' : 'husband'}
```

Comment:
Let's move the check for sex to a variable and place it before return

Then you can do the check for `isMarried` inside the `p`